STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/DownloadLimit.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Infomaniak SwissTransfer - Multiplatform
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.infomaniak.multiplatform_swisstransfer.common.models
+
+enum class DownloadLimit(val value: String) {
+    TWOHUNDREDFIFTY("250"),
+    ONEHUNDRED("100"),
+    TWENTY("20"),
+    ONE("1");
```

Comment:
Does it really have to be thongs? Is INT not enough?

Reply:
If i'm putting this as integer, I'll have to do a `toString` on the android side because I'm using this value during display.

Reply:
Okay i understand